### PR TITLE
Travis updates

### PR DESCRIPTION
**Pull request by TravisSperry**
This includes updates to database configuration for Travis CI. Build still fails because current test suite fails, but if test suite is completed and passing build, will pass.

### CODE REVIEW

**Comment by degoeym** [comment]
Well I guess this invalidates the need for my code fix for #32....

**Comment by TravisSperry** [comment]
Good to merge? This ads modifications to @degoeym center styling and updates to travis.yml file

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,13 @@
 language: ruby
 rvm:
   - 2.3.1
+<<<<<<< HEAD
```
Comment (cartoloupe) [inline .travis.yml]: it looks like merge conflict notation lines are still in here

**Comment by cartoloupe** [comment]
Looks good to merge, once merge conflicts in `.travis.yml` are clarified 👍

**Pull request merged**